### PR TITLE
Input and list

### PR DESCRIPTION
**Pull request by iamalhera**

### CODE REVIEW

File: src/components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+.btn-primary{
+    width : 100% ;
+}
+.form-signup{
+    padding : 1.5rem ;
+    border :2px solid var(--ca-primary-color-lighter);
+    background-color: var(--white-color);
+    margin : 1rem ;
+    width : 350px ;
```
Comment (sri-rishi) [inline src/components/input/input.css:30]: can used rem instead of px

File: src/components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    border-radius: 5px;
+    font-family: 'Montserrat', sans-serif;
+
+}
+.form-header{
+    font-size : 2.0rem ;
```
Comment (sri-rishi) [inline src/components/input/input.css:36]: just give 2rem not 2.0rem

**Review by sri-rishi — COMMENTED** [review]
Otherwise code  looks good.

File: src/components/list/list.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+}
+.type-circle{
+list-style-type:circle
+}
+
+.container-column{
+    align-items: center;
+}
+.stacked-list-item{
+        max-width: 500px;
```
Comment (nikhiljugale007) [inline src/components/list/list.css:34]: User rems.

**Review by nikhiljugale007 — COMMENTED** [review]
UI suggestion : 
You can give some padding for input, 
In stacked list items are not aligned properly

File: src/components/input/input.css (hunk 1 above)
Comment (iamalhera) [reply src/components/input/input.css:30]: i wanted to make a standard signup form, so did that

Comment (iamalhera) [reply src/components/input/input.css:30]: but it is still reponsive

File: src/components/list/list.css (hunk 3 above)
Comment (iamalhera) [reply src/components/list/list.css:34]: fixed it

**Pull request merged**